build.gradle

```diff
@@ -40,7 +40,6 @@ dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     compile "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
     compile "com.nhaarman:mockito-kotlin-kt1.1:$mockito_version"
-    testCompile 'junit:junit:4.12'

```

Comment:
Do tests still run without the JUnit dependency?

Reply:
Yes. The tests use Spek as a test runner. The all the builds in the PR have passed.